src/aes.h

```diff
@@ -24,13 +24,20 @@
 
 namespace toit {
 
-class AesCbcContext : public SimpleResource {
+class AesEcbContext : public SimpleResource {
  public:
-  TAG(AesCbcContext);
-  AesCbcContext(SimpleResourceGroup* group, const uint8* key, const uint8* iv, bool encrypt);
-  ~AesCbcContext();
+  TAG(AesEcbContext);
+  AesEcbContext(SimpleResourceGroup* group, const Blob* key, bool encrypt);
+  virtual ~AesEcbContext();
 
   mbedtls_aes_context context_;
+};
+
+class AesCbcContext : public AesEcbContext {
+ public:
+  TAG(AesCbcContext);
+  AesCbcContext(SimpleResourceGroup* group, const Blob* key, const uint8* iv, bool encrypt);
+
   uint8 iv_[16];
```

Comment:
Should we have a named constant for 16?

Reply:
I put it in the AesContext class and called it AES_BLOCK_SIZE to match the AES block size and name.